tests/test_globalconstraints.py

```diff
@@ -2,7 +2,7 @@
 import unittest
 import cpmpy as cp
 from cpmpy.expressions.globalconstraints import GlobalConstraint
-from cpmpy.expressions.globalfunctions import GlobalFunction
+from cpmpy.expressions.globalfunctions import GlobalFunction, Abs
```

Comment:
For the rest of the global constraints and global functions we do not import them and just use cp.function, e.g. cp.max. To be consistent, in this case we should not import abs (it is weird to be the only one imported) and just use cp.abs